src/test/java/uk/gov/hmcts/cmc/claimstore/services/ResponseDeadlineCalculatorTest.java

```diff
@@ -25,91 +17,31 @@
-
-        assertThat(responseDeadline).isWeekday().isTheSame(toDate("2017-05-30"));
-    }
-
-    @Test
-    public void calculateResponseDeadlineWhenResponseDateOnGoodFridayThenReturnFirstWorkingDay() {
-        LocalDate mondayBeforeGoodFriday = toDate("2017-03-26");
-        LocalDate responseDeadline = calculator.calculateResponseDeadline(mondayBeforeGoodFriday);
-
-        assertThat(responseDeadline).isWeekday().isTheSame(toDate("2017-04-18"));
+    public void setUp() {
+        calculator = new ResponseDeadlineCalculator(DAYS_FOR_SERVICE, DAYS_FOR_RESPONSE, POSTPONE_BY);
     }
 
     @Test
-    public void calculateWhenDeadlineIsOnBankHolidaysAndPostponedDeadlineIsOnWorkingDay() {
+    public void calculateResponseDeadline() {
+        LocalDate issuedOn = now();
+        LocalDate expectedDeadlineDate = issuedOn.plusDays(DAYS_FOR_RESPONSE + DAYS_FOR_SERVICE);
 
-        LocalDate issuedOn = toDate("2017-05-08");
         LocalDate responseDeadline = calculator.calculateResponseDeadline(issuedOn);
-        LocalDate postponedDeadline = calculator.calculatePostponedResponseDeadline(issuedOn);
-        LocalDate expectedDeadlineDate = toDate("2017-05-30"); // Tue as Mon was Bank holidays
-        LocalDate expectedPostponedDate = toDate("2017-06-12"); // Mon as we calculate it based on issue date
-
-        assertThat(responseDeadline).isWeekday().isTheSame(expectedDeadlineDate);
-        assertThat(postponedDeadline).isWeekday().isTheSame(expectedPostponedDate);
-    }
-
-    @Test
-    public void calculateWhenDeadlineIsWorkingDayAndPostponedDeadlineIsOnWorkingDay() {
-        LocalDate issuedOn = toDate("2017-09-18");
-        LocalDate expectedDeadlineDate = toDate("2017-10-09");
-        LocalDate expectedPostponedDate = toDate("2017-10-23");
-
-        LocalDate responseDeadline = calculator.calculateResponseDeadline(issuedOn);
-        LocalDate postponedDeadline = calculator.calculatePostponedResponseDeadline(issuedOn);
 
         assertThat(responseDeadline).isWeekday().isTheSame(expectedDeadlineDate);
-        assertThat(postponedDeadline).isWeekday().isTheSame(expectedPostponedDate);
     }
 
     @Test
-    public void calculateWhenDeadlineIsOnWeekDayAsWellAsPostponedDeadline() {
-        LocalDate issuedOn = toDate("2017-09-15");
-        LocalDate expectedDeadlineDate = toDate("2017-10-04");
-        LocalDate expectedPostponedDate = toDate("2017-10-18");
+    public void calculatePostponedResponseDeadline() {
+        LocalDate issuedOn = now();
+        LocalDate expectedPostponedDate = issuedOn.plusDays(DAYS_FOR_RESPONSE + DAYS_FOR_SERVICE + POSTPONE_BY);
 
-        LocalDate responseDeadline = calculator.calculateResponseDeadline(issuedOn);
         LocalDate postponedDeadline = calculator.calculatePostponedResponseDeadline(issuedOn);
 
-        assertThat(responseDeadline).isWeekday().isTheSame(expectedDeadlineDate);
         assertThat(postponedDeadline).isWeekday().isTheSame(expectedPostponedDate);
```

Comment:
Please remove use of `.isWeekday()` method. It is no longer requirement to have a weekday date and it would fail on weekends. Please do the same in other place.

Reply:
done